recipes/symengine/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.1.0.git" %}
+
+package:
+  name: symengine
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/symengine/symengine
+  git_tag: 3f58acc1246661323b1daf2adc3f9fa6a8762df4
```

Comment:
Pleas do the same as above. (Also, can you ping the devs and request an official release?)